### PR TITLE
CVE-2017-9798

### PR DESCRIPTION
**Pull request by JO4IVIV4**
Edited CVE-2017-9798 for Apache httpd

### CODE REVIEW

File: cves/CVE-2017-9798.yml [hunk 1]
```diff
@@ -121,10 +126,12 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer:
-  date:
-  automated:
-  google:
+  answer: |
+    This vulnerability was found by two people. Hanno Böck on September 18, 2017 and
+    Jon King of OPNAV N1, Giulio Comi of Horizon Security on Oct 16, 2018
```
Comment (akn4743) [inline cves/CVE-2017-9798.yml:131]: I'm a bit confused. The patch to fix this bug was in October 2017 but it was found by someone else in October 2018. Was the bug reintroduced?

Comment (JO4IVIV4) [reply cves/CVE-2017-9798.yml:131]:
```Author: William A. Rowe Jr <wrowe@apache.org>
Date:   Tue Feb 26 17:35:18 2019 +0000

    Simplify charset handling

    git-svn-id: https://svn.apache.org/repos/asf/httpd/httpd/trunk@1854409 13f79535-47bb-0310-9956-ffa450edef68
```
the actual fix commit was in 2019

File: cves/CVE-2017-9798.yml [hunk 2]
```diff
@@ -48,7 +48,12 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  users are able to corrupt the global methods table by setting the Limit directive
```
Comment (PaceyLong) [inline cves/CVE-2017-9798.yml]: Users* capital U

File: cves/CVE-2017-9798.yml [hunk 3]
```diff
@@ -193,8 +200,11 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: | 
+      this vulnerability can be exploited by users because they are able to
```
Comment (PaceyLong) [inline cves/CVE-2017-9798.yml]: This* capital T

File: cves/CVE-2017-9798.yml (hunk 1 above)
Comment (PaceyLong) [inline cves/CVE-2017-9798.yml:131]: Is there any mention how this vulnerability was found?

File: cves/CVE-2017-9798.yml [hunk 4]
```diff
@@ -134,8 +141,8 @@ subsystem:
     Look at the path of the source code files code that were fixed to get
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged.
-  answer:
-  name:
+  answer: Server
```
Comment (PaceyLong) [inline cves/CVE-2017-9798.yml]: Please write a full sentence.

File: cves/CVE-2017-9798.yml (hunk 1 above)
Comment (JO4IVIV4) [reply cves/CVE-2017-9798.yml:131]: There was no mention.

File: cves/CVE-2017-9798.yml (hunk 2 above)
Comment (JO4IVIV4) [reply cves/CVE-2017-9798.yml]: fixed

File: cves/CVE-2017-9798.yml (hunk 3 above)
Comment (JO4IVIV4) [reply cves/CVE-2017-9798.yml]: fixed

File: cves/CVE-2017-9798.yml (hunk 4 above)
Comment (JO4IVIV4) [reply cves/CVE-2017-9798.yml]: fixed

**Pull request merged**